runtime/doc/usr_26.txt

```diff
@@ -81,23 +81,6 @@ Move the cursor to "1." and type: >
 The "1." will change to "4.".  Again, you can use "." to repeat this on the
 other numbers.
 
-Another example:
-
-	006	foo bar ~
-	007	foo bar ~
-
-Using CTRL-A on these numbers results in:
-
-	007	foo bar ~
-	010	foo bar ~
-
-7 plus one is 10?  What happened here is that Vim recognized "007" as an octal
-number, because there is a leading zero.  This notation is often used in C
-programs.  If you do not want a number with leading zeros to be handled as
-octal, use this: >
-
-	:set nrformats-=octal
```

Comment:
Sorry if I missed the discussion of this, but should we keep this help section and instead just mention `:set nrformats+=octal` at the beginning?